From db8ef1380b4e2fb9f52552408354ccc878cc2304 Mon Sep 17 00:00:00 2001
From: Jason Wang <jasowang@redhat.com>
Date: Mon, 29 Sep 2014 00:59:10 -0400
Subject: [virt] hyperv: fix apparent cut-n-paste error in send path teardown

Message-id: <1411952358-34104-27-git-send-email-jasowang@redhat.com>
Patchwork-id: 96778
O-Subject: [RHEL7.1 PATCH 26/34] hyperv: fix apparent cut-n-paste error in send path teardown
Bugzilla: 1146357
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Vitaly Kuznetsov <vkuznets@redhat.com>

From: Dave Jones <davej@redhat.com>

Bugzilla: 1146357

c25aaf814a63: "hyperv: Enable sendbuf mechanism on the send path" added
some teardown code that looks like it was copied from the recieve path
above, but missed a variable name replacement.

Signed-off-by: Dave Jones <davej@redhat.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit 2f18423d7ebf8044a9edaca840be5ae199fb32f6)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/hyperv/netvsc.c b/drivers/net/hyperv/netvsc.c
index f4aebab..f7647ef 100644
--- a/drivers/net/hyperv/netvsc.c
+++ b/drivers/net/hyperv/netvsc.c
@@ -190,7 +190,7 @@ static int netvsc_destroy_buf(struct netvsc_device *net_device)
        "unable to teardown send buffer's gpadl\n");
    return ret;
   }
-  net_device->recv_buf_gpadl_handle = 0;
+  net_device->send_buf_gpadl_handle = 0;
  }
  if (net_device->send_buf) {
   /* Free up the receive buffer */
-- 
1.7.1